client/src/components/Carousel/Carousel.jsx

```diff
@@ -47,23 +47,38 @@ export default class Carousel extends React.Component {
   }
 }
 
-// Carousel.defaultProps = {
-//   header: null,
-//   features: null,
-//   updateActive: null,
-//   activeIndex: 0,
-// };
+Carousel.defaultProps = {
+  products: [],
+  updateProduct: null,
+};
 
-// Carousel.propTypes = {
-//   features: PropTypes.arrayOf(
-//     PropTypes.shape({
-//       heading: PropTypes.string,
-//       description: PropTypes.string,
-//       posX: PropTypes.number,
-//       posY: PropTypes.number,
-//     }),
-//   ),
-//   header: PropTypes.string,
-//   updateActive: PropTypes.func,
-//   activeIndex: PropTypes.number,
-// };
+Carousel.propTypes = {
```

Comment:
I see you are fleshing out the data details. Looking forward to seeing it in action.